### ISSUE TITLE
No licence for NPM code

### ISSUE THREAD

**Issue by nedrichards** [flatpak/flatpak-builder-tools#10]
There isn't a licence on the NPM code. It'd be good if we had one (pip is GPLv3+ but there's no requirement to be in sync) and now, whilst we only have two contributors would be the best time.

@alexlarsson @adityashah1212 what licence do you want?

**Comment by alexlarsson** [comment]
Bunch of npm code is from @gicmo too.

This is a basically script, and as such its likely to be copied about and reused, so IMHO a very lenient license like MIT makes a lot of sense.

**Comment by gicmo** [comment]
Sounds good to me!

**Comment by nedrichards** [comment]
Yeah, I'd be pro MIT ideally avoid Apache 2 since we already have some GPL3 code in the repo and things could get annoying.

**Comment by TingPing** [comment]
I can re-license mine to MIT, as mentioned it is fairly trivial.

**Comment by alexlarsson** [comment]
So, lets go MIT?

**Comment by TingPing** [comment]
Sure.

@ptomato Ok with relicensing your pip patches to MIT?

**Comment by ptomato** [comment]
:+1: No problem.
